presto-main/src/main/java/com/facebook/presto/execution/QueryStateMachine.java

```diff
@@ -365,6 +367,7 @@ public QueryInfo getQueryInfo(Optional<StageInfo> rootStage)
             }
             completeInfo = completeInfo && stageInfo.isCompleteInfo();
             operatorStatsSummary.addAll(stageInfo.getStageStats().getOperatorSummaries());
+            spilledBytes += stageStats.getSpilledDataSize().toBytes();
```

Comment:
Move this to somewhere around L350 (so that it's close to other similar stats) or 368 (right after outputPositions)